SFR/Assembly/inner_assembly/make_SFR_assembly_model.py

```diff
@@ -0,0 +1,209 @@
+import openmc 
+import numpy as np
+from argparse import ArgumentParser
+import openmc.material 
+import warnings
+warnings.filterwarnings('ignore')
+
+
+# Create the argument parser
+ap = ArgumentParser(description="SFR Pincell Model Generator")
+ap.add_argument('-n', dest='n_axial', type=int, default=100, help='Number of cells in the Z direction')
```

Comment:
How about a default of 1? I think that makes sense as a default because that would be the best performance choice if not needing extra cells for depletion/feedback.